app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -377,7 +512,15 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_configChange_sectionDividerIsNotDisplayed() {
+  fun testRecentlyPlayedTestActivity_configChange_sectionDividerIsNotDisplayed() { // ktlint-disable max-line-length
+    storyProgressTestHelper.markRecentlyPlayedForFractionsStory0Exploration0(
```

Comment:
/ ktlint-disable max-line-length 
not required

Reply:
Done.